### ISSUE TITLE
Rubicon adapter issue

### ISSUE THREAD

**Issue by ericdevries** [prebid/Prebid.js#270]
Hello,

I am trying to use the rubicon adapter, but it fails on the following 3 lines. The bids variable is supposed to be an array

```
// line 276 to 278, adapters/rubicon.js
for (var key in bids) {
  slots.push(_defineSlot(bids[key]));
}
```

But it loops it as if its an object. Changing this a for (var i = bids.length; etc) or a bids.forEach loops make it work again.

**Comment by protonate** [comment]
@bretg could you have a look at this?

**Comment by bretg** [comment]
We're looking into this issue this week.

**Comment by bretg** [comment]
@protonate - pull request https://github.com/prebid/Prebid.js/pull/301

**Comment by protonate** [comment]
Released with [0.8.0](https://github.com/prebid/Prebid.js/releases/tag/0.8.0)
